src/tests/cpp/fwklib/Timer.hpp

```diff
@@ -28,18 +28,18 @@
 
 #ifdef _WIN32
 
-#include <time.h>
-#include <stdio.h>
-#include <stdlib.h>
+#include <ctime>
+#include <cstdio>
+#include <cstdlib>
 #include <windows.h>
 #include <winbase.h>
 
 #endif
 
 #ifdef _SOLARIS
 
-#include <time.h>
-#include <stdio.h>
+#include <ctime>
```

Comment:
I bet this could be cleaned up by moving time, cstudio, cstdlib out of the conditionals since all platforms have those includes now.